changelog.d/5733.misc

```diff
@@ -0,0 +1 @@
+Communities/Groups are removed completely
```

Comment:
Please add another file `5733.sdk` to notify SDK users about the deleted/updated API(s).